scripts/projectView.js

```diff
@@ -12,7 +12,7 @@
 
   projectView.initIndexPage = function(){
     Project.all.forEach(function(a){
-      $('#projects').append(a.toHtml());
+      $('#projects').append(a.toHtml('project'));
     });
```

Comment:
Why is that a and not a semantic name? Because this is you copying, that's why. call it project instead.